### ISSUE TITLE
Dependency not found when installing from pypi

### ISSUE THREAD

**Issue by fstab50** [fstab50/libtools#1]
* ``Pygments`` library is not found when installing from the python global pkg registry.
* Installing locally from source is successful; albeit bc `requirements.txt` file preinstalls Pygments dep into the virtual env

**Comment by fstab50** [comment]
Output when installing from testpypi or pypi:

```
Collecting curses-menu (from -r /home/blake/git/keyup/requirements.txt (line 4))
  Using cached https://files.pythonhosted.org/packages/3b/7d/82f1492afd33f92e64d44d51f65eff4c39382642cfced65d5748a770bb8f/curses-menu-0.5.0.zip
Collecting libtools (from -r /home/blake/git/keyup/requirements.txt (line 5))
  Using cached https://files.pythonhosted.org/packages/a4/4a/64557eeb0a171d909044682e60a48c8ff578f5eb118039d00eae23c3f913/libtools-0.1.8.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-86bq69gz/libtools/setup.py", line 30, in <module>
        import libtools
      File "/tmp/pip-install-86bq69gz/libtools/libtools/__init__.py", line 44, in <module>
        from libtools.export import export_json_object
      File "/tmp/pip-install-86bq69gz/libtools/libtools/export.py", line 4, in <module>
        from pygments import highlight, lexers, formatters
    ModuleNotFoundError: No module named 'pygments'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-86bq69gz/libtools/
```

**Comment by fstab50** [comment]
Fixed in v0.1.10 by moving `export.py` to io subarea.
